src/lib/ApxData/Sharing/Share.cpp

```diff
@@ -94,6 +94,9 @@ void Share::exportTriggered()
 
     QFileInfo fi(dlg.selectedFiles().first());
 
+    QSettings().setValue(QString("DefaultExportPath"), dlg.directory().absolutePath());
```

Comment:
You should prepend setting name with file format name so that each exporter will have its own export directory.

`QSettings().setValue(QString("DefaultExportPath_%1").arg(fmt),...`